### PR TITLE
[Fix #1485] PdrDashboard v0.4 - Feeds page metrics

### LINKED ISSUE #1485: [Pdr Dashboard] v0.4 - Display main metrics section
Under the menu display a row containing main metrics over all feeds, such as: Volume, Feeds, Sales, Revenue, Accuracy(avg)

### PR DESCRIPTION
**Pull request by calina-c**

### CODE REVIEW

File: pdr_backend/pdr_dashboard/dash_components/view_elements.py [hunk 1]
```diff
@@ -27,6 +27,17 @@ def get_information_text(tooltip_id: str):
         case "tooltip-switch-feeds":
             return """Toggle this switch to view only the feeds associated with
                 the selected predictoors."""
+        case "tooltip-feeds_page_Volume_metric":
+            return "Total stake placed on predictions"
+        case "tooltip-feeds_page_Feeds_metric":
+            return "Total number of feeds"
+        case "tooltip-feeds_page_Sales_metric":
+            return "Number of subscriptions to feeds"
```
Comment (KatunaNorbert) [inline pdr_backend/pdr_dashboard/dash_components/view_elements.py:35]: Rename to: "Number of subscriptions purchased across all feeds"

File: pdr_backend/pdr_dashboard/dash_components/util.py [hunk 2]
```diff
@@ -260,6 +267,47 @@ def get_predictoors_data_from_payouts(
...
+    )
+
+    feeds = feeds[0] if feeds else 0
+
+    accuracy, revenue, volume = _query_db(
+        lake_dir,
+        f"""
+            SELECT
+                SUM(CASE WHEN payout > 0 THEN 1 ELSE 0 END) * 100.0 / COUNT(*) AS avg_accuracy,
+                SUM(payout) as total_revenue,
```
Comment (KatunaNorbert) [inline pdr_backend/pdr_dashboard/dash_components/util.py:288]: The revenue should be calculated from subscription sales, how much revenue the feeds has brought from subscription sells

**Review by KatunaNorbert — CHANGES_REQUESTED** [review]

File: pdr_backend/pdr_dashboard/pages/feeds.py [hunk 3]
```diff
@@ -28,6 +31,29 @@ def layout(self):
...
+            children=[
+                get_metric(
+                    label=key,
+                    value=value,
+                    value_id=f"feeds_page_{key}_metric",
+                )
+                for key, value in stats.items()
+            ],
+            style={
+                "display": "flex",
```
Comment (kdetry) [inline pdr_backend/pdr_dashboard/pages/feeds.py:47]: we can put Div styles to the CSS file. We can use the div ID

**Review by kdetry — COMMENTED** [review]
I have some suggestions about the code

**Review by KatunaNorbert — APPROVED** [review]

**Pull request merged**